website/mailinglists/services.py

```diff
@@ -55,6 +55,12 @@ def get_automatic_lists():
     )
     mentors = _get_members_email_addresses([x.member for x in active_mentorships])
 
+    oldmembers = [
+        m
+        for m in Member.objects.all()
+        if m.current_membership.type != Membership.MEMBER and m.has_been_member()
```

Comment:
As Job mentioned, this will likely throw an error when there are Members without a current_membership.